finagle/buoyant/src/main/scala/com/twitter/finagle/buoyant/SocketOptionsConfig.scala

```diff
@@ -0,0 +1,14 @@
+package com.twitter.finagle.buoyant
+
+import com.twitter.finagle.Stack
+import com.twitter.finagle.transport.Transport
+
+case class SocketOptionsConfig(
+  disableTcpNoDelay: Boolean = true,
```

Comment:
I think the name of this parameter is backwards.  It maps to the finagle argument `noDelay`.

More generally, I think we should remove the `enabled` or `disable` words and just make all of these positive:

`tcpNoDelay = true, resuseAddr = true, reusePort = false`

Reply:
Yea, good point. I was running through a couple of different versions to see what names would describe the options well. I do agree that `disable-` makes the word more negative.